db/migrate/201610012033_create_watermarks.rb

```diff
@@ -0,0 +1,12 @@
+class CreateWatermarks < ActiveRecord::Migration
+  def change
+    create_table :watermarks do |t|
+      t.integer   :user_id, null: false
+      t.string    :track_id, null: false
+      t.string    :slug, null: false
```

Comment:
Oh right... I was thinking only in the `mark all as read` for an entire track. Thanks!